src/app.ts

```diff
@@ -1,6 +1,87 @@
 
 const startApp = async () => {
+    const tbody = document.querySelector('[data-sink]')
+    const tr = tbody?.querySelectorAll('tr')
+    const prevBtn = document.querySelector('[data-prevbtn')
```

Comment:
Codacy has a fix for the issue: [Replace `····const·prevBtn·=·document.querySelector('[data-prevbtn')` with `··const·prevBtn·=·document.querySelector("[data-prevbtn");`](https://app.codacy.com/gh/alic-xc/Test/pullRequest?prid=9965044)
```suggestion
  const prevBtn = document.querySelector("[data-prevbtn");
```